pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -4906,6 +4906,15 @@
         "4.0.4.0": "4.6.0"
       }
     },
+    "System.Text.Json": {
```

Comment:
Are we going to have a portable package for this?

Reply:
> These APIs will be targeting **.NET Core specifically** as part of v1.
Providing a netstandard implementation can be considered in the future.

We will gather feedback from users and take that into account when deciding if we should have a portable package.